src/SlashCommands.tsx

```diff
@@ -30,7 +30,7 @@ import { SlashCommand as SlashCommandEvent } from "@matrix-org/analytics-events/
 
 import { MatrixClientPeg } from "./MatrixClientPeg";
 import dis from "./dispatcher/dispatcher";
-import { _t, _td, ITranslatableError, newTranslatableError } from "./languageHandler";
+import { _t, _td, UserFriendlyError } from "./languageHandler";
```

Comment:
SonarCloud coverage complaint and most is from this file, https://github.com/matrix-org/matrix-react-sdk/pull/10440/checks?check_run_id=12312160706

Adding test coverage here is pretty tangent to the this PR so I'm going to leave it as-is.